### PR TITLE
Build for Windows

### PR DESCRIPTION
**Pull request by jakirkham**
Makes an attempt at getting this to build on Windows.

cc @FrancescElies

### CODE REVIEW

**Comment by jakirkham** [comment]
Could you please review this @msarahan ?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,20 +9,26 @@ source:
...
   build:
+    - python       # [win]
     - toolchain
     - cmake
+    - snappy       # [unix]
     - zlib 1.2.*
-    - snappy
   run:
-    - snappy
+    - snappy       # [unix]
```
Comment (jakirkham) [inline recipe/meta.yaml]: No `snappy` on Windows yet. Sorry. 😞 Maybe one day... ( https://github.com/google/snappy/pull/16 )

Comment (jakirkham) [inline recipe/meta.yaml]: No `snappy` on Windows yet. Sorry. 😞 Maybe one day... ( https://github.com/google/snappy/pull/16 )

**Comment by jakirkham** [comment]
cc @mrocklin

**Comment by jakirkham** [comment]
Seems like Python 2.7 64-bit is a bit pesky. If anyone has suggestions, would definitely appreciate guidance there.

**Comment by jakirkham** [comment]
Looks like we may need the VS 2008 64-bit hack to make this work. Will give a go at integrating that into `conda-forge-build-setup`. Depending on how that goes, we may want to decide if some `blosc` support on Windows is better than no `blosc` support on Windows.

**Comment by mrocklin** [comment]
I'm very glad to see this happen.  Thank you for your tireless work here @jakirkham !

**Comment by jakirkham** [comment]
Ok, I'm at a bit of a loss. Something is going terribly wrong with VS 2008 64-bit and I don't think I'm the one to fix it. This means that Python 2.7 64-bit is out near term. That being said, all of the other builds are fine.

For now, I'm going to back out the hack and disable the VS 2008 64-bit build. Even though this doesn't build everything, this still builds quite a bit and is a vast improvement on the previous Windows situation. As such, it should still be mergeable.

Will write up an issue for VS 2008 64-bit support.

**Comment by jakirkham** [comment]
Alright, I think this is good enough on my end.

As noted, I have pulled out VS 2008 64-bit for the near term as that looks to be quite [problematic]( https://ci.appveyor.com/project/conda-forge/blosc-feedstock/build/1.0.8/job/ut95t0c1a817rp9o ) (even for [upstream]( https://github.com/Blosc/c-blosc/commit/9f0c10ce170f1108065c9bb4ac39730b31694a10 )). Everything else (even VS 2008 32-bit) seems to be working fine. Maybe we could use the VS 2008 32-bit build in place of a proper 64-bit build near term?

Would be great if a seasoned Windows developer like @msarahan or @mingwandroid could review.

**Comment by mingwandroid** [comment]
I'll definitely get round to your review requests soon @jakirkham, but not today. The mammoth task that was rstudio is coming to an end.

**Comment by jakirkham** [comment]
Not a problem. Sorry to ping you so frequently.

**Comment by jakirkham** [comment]
Seems the linter posted the same message twice. 😕 Pretty sure this issue was fixed long ago and has tests. Any ideas @pelson ?

**Comment by jakirkham** [comment]
Should be fixed with PR ( https://github.com/conda-forge/blosc-feedstock/pull/17 ). Builds pending.

**Pull request closed without merging**